samples/bot-types-of-cards/nodejs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "teams-bot",
+    "version": "1.0.0",
+    "description": "Bot Builder v4 Bot Teams sample",
```

Comment:
WHy ?
Bot Builder v4 Bot Teams sample

Reply:
Fixed and changes to  : Types of cards using bot